app/src/gui/components/notebook/index.tsx

```diff
@@ -172,16 +215,16 @@ export default function NotebookComponent(props: NotebookComponentProps) {
                 textColor="inherit"
                 variant="scrollable"
                 scrollButtons="auto"
-                // centered={mq_above_md ? false : true}
               >
                 <Tab label="Records" {...a11yProps(0, NOTEBOOK_NAME)} />
-                <Tab label="Info" {...a11yProps(1, NOTEBOOK_NAME)} />
-                <Tab label="Settings" {...a11yProps(2, NOTEBOOK_NAME)} />
+                <Tab label="Details" {...a11yProps(1, NOTEBOOK_NAME)} />
+                <Tab label="Access" {...a11yProps(2, NOTEBOOK_NAME)} />
+                <Tab label="Layers" {...a11yProps(3, NOTEBOOK_NAME)} />
```

Comment:
'Access' and 'Layers' don't contain anything - just confusing at this point. Please remove them until we have something useful to put in there.